aws-lambda/README.md

```diff
@@ -0,0 +1,101 @@
+# ![](./img/integration_awslambda.png) Amazon Lambda
+
+- [Description](#description)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Metrics](#metrics)
+- [License](#license)
+
+### DESCRIPTION
+
+Use SignalFx to monitor Lambda via [Amazon CloudWatch](https://github.com/signalfx/integrations/tree/master/aws)[](sfx_link:aws).
+
+#### FEATURES
+
+##### Built-in dashboards
+
+- **Lambda Overview Cloudwatch**: Overview of all data from Lambda via CloudWatch.
+
+  [<img src='./img/dashboard_lambda_overview_cloudwatch.png' width=200px>](./img/dashboard_lambda_overview_cloudwatch.png)
+
+- **Lambda Overview SignalFx Wrapper**: Overview of all data from Lambda via SignalFx Wrapper.
+
+  [<img src='./img/dashboard_lambda_overview_cloudwatch.png' width=200px>](./img/dashboard_lambda_overview_wrapper.png)
+  
+- **Lambda Instance Cloudwatch**: Instance view for specific Lambda function via CloudWatch.
+
+  [<img src='./img/dashboard_lambda_instance_cloudwatch.png' width=200px>](./img/dashboard_lambda_instance_cloudwatch.png)
+  
+- **Lambda Instance Cloudwatch**: Instance view for specific Lambda function via SignalFx Wrapper.
+
+  [<img src='./img/dashboard_lambda_instance_wrapper.png' width=200px>](./img/dashboard_lambda_instance_wrapper.png)
+
+### INSTALLATION
+
+SignalFx provides two methods of monitoring Lambda metrics. SignalFx Wrapper provides real-time invocations, errors and cold starts while CloudWatch provides an accurate durations. 
+
+#### SignalFx Wrapper
+
+SignalFx Wrapper provides real time monitoring of lambda functions as well as ability to send custom metric from your applications to SignalFx.
```

Comment:
as well as ability to send custom metric  ==> as well as the ability to send custom metrics